modules/travel_pay/app/models/travel_pay/common_carrier_expense.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative '../../../lib/travel_pay/constants'
+
+module TravelPay
+  class CommonCarrierExpense < BaseExpense
+    attribute :reason_not_using_pov, :string
```

Comment:
Suggestion: A comment above this line explaining that `pov` means "privately owned vehicle"